firmware/application/ui/ui_alphanum.hpp

```diff
@@ -43,14 +43,14 @@ class AlphanumView : public TextEntryView {
     bool on_encoder(const EncoderEvent delta) override;
 
    private:
-    const char* const keys_upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ, .<";
-    const char* const keys_lower = "abcdefghijklmnopqrstuvwxyz, .<";
-    const char* const keys_digit = "0123456789!\"#'()*+-/:;=>?@[\\]<";
+    const char* const keys_upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ, ._";
+    const char* const keys_lower = "abcdefghijklmnopqrstuvwxyz, ._";
+    const char* const keys_digit = "0123456789!\"#'()*+-/:;=<>@[\\]?";
 
     const std::pair<std::string, const char*> key_sets[3] = {
-        {"Upper", keys_upper},
-        {"Lower", keys_lower},
-        {"Digit", keys_digit}};
+        {"ABC", keys_upper},
+        {"abc", keys_lower},
+        {"123.", keys_digit}};
```

Comment:
nit: I'd prefer `123` without the dot so the lengths are consistent.

Reply:
Thank you! Originally i thought the dot meant marks, but we can discuss about this. Maybe I should delete it.

Reply:
It was called "Digits" before and no one was wondering where the symbols were. ;)
I think you can remove it.

Reply:
OOOH haha ya, thank you pointing it out, will remove it. XD

Reply:
removed, thank you!